ansible/roles/cloudflared/tasks/main.yml

```diff
@@ -17,21 +17,23 @@
   git:
     repo: https://github.com/xUnholy/cloudflared.git
     dest: cloudflared
+    version: master
 
 - name: Compile cloudflared binary
   command: make cloudflared
```

Comment:
We might be able to use the ansible `make` module: https://docs.ansible.com/ansible/latest/modules/make_module.html
Havent used that module before though, so no clue if it actually works properly.

Reply:
Updating it per the documentation you provided